RPLCD/i2c.py

```diff
@@ -77,7 +110,16 @@ def __init__(self, address, port=1,
 
     def _init_connection(self):
         self.bus = SMBus(self.port)
-        c.msleep(50)
+
+        if(self.i2c_expander == "PCF8574"):
+            c.msleep(50)
+        elif(self.i2c_expander == "MCP23008"):
+            # Set IO DIRection to output on all GPIOs (GP0-GP7)
+            self.bus.write_byte_data(self.address, c.MCP23008_IODIR, 0x00)
```

Comment:
Its not a constant. 0x00 is sent to the IODIR register, one bit for each gpio on the expander.
For example sending 0b10000010 would set gpio 7 and 1 to input (1) and the rest to output (0).
Don't think that setting it to a constant would make it easier to read the code. It's only used once and will not change.